src/services/blog/index.js

```diff
@@ -1,11 +1,11 @@
-import { getTheDevRead } from './request.js';
-import redisClient from '#utils/redis.js';
-import { randomArray } from '#utils/random.js';
-import { renderTemplate } from '#utils/template.js';
-import { getCommandArgs } from '#utils/command.js';
-import { logger } from '#utils/logger/logtail.js';
+import { getTheDevRead } from "./request.js";
```

Comment:
file ini direname jadi `devread` aja deh, bikin bingung kalo `blog`

Reply:
ok later this midnight ya